cli/tests/resources/test.csv

```diff
@@ -0,0 +1,2 @@
+procedure_id,mrn,accession_number,project_name,omop-es-datetime

```

Comment:
Do we want to require the real study date as well? Only for ordering, but that might be reasonable?